source/PlayState.hx

```diff
@@ -42,34 +61,32 @@ class PlayState extends FlxState
 		var baseObstacleParameters = new ObstacleParameters(FlxG.width, FlxG.height, 200, 10, 50);
 		var obstacleVariation = new ObstacleVariation(-1, .9, 1, 2);
 
-		obstacleGenerator = new ObstacleGenerator<Obstacle>(SECONDS_PER_OBSTACLE,
-			baseObstacleParameters, obstacleVariation, generatedObstacles);
+		obstacleGenerator = new ObstacleGenerator<Obstacle>(SECONDS_PER_OBSTACLE, baseObstacleParameters, obstacleVariation, generatedObstacles);
 		add(obstacleGenerator.obstacles);
 	}
 
-	private function setUpDeadlyObstacles()
-	{
+	private function setUpDeadlyObstacles() {
```

Comment:
These formatting changes might cause some merge conflict headaches for some folks. It's tricky to avoid when everybody's got a formatting file of their own that automatically updates things when they save.

This is more of a problem of me not having a consistent style guide for folks to follow + a formatting file that makes it easy to use with modern editors.